.travis.yml

```diff
@@ -59,6 +63,7 @@ matrix:
       env: TARGET=go
 
 before_install:
+  - ./.travis/before-install-mvn3.3.9-trusty.sh
```

Comment:
Hi 
the point of the before-install-$TRAVIS_OS_NAME-$TARGET.sh is precisely to avoid polluting every target with target specific stuff
please move this into before-install-linux-dotnet.sh